src/components/common/Layout.tsx

```diff
@@ -9,6 +9,24 @@ import Header from "./Header";
  * place in <head> through React Helmet.
  */
 export default class Layout extends React.Component<LayoutProps> {
+  title: string;
+  description: string;
+
+  /**
+   * Initialize the component and its state.
```

Comment:
nitpicking but- this one is stateless 😄 (in the sense of, does not have mutable state and only receives data from its parent) so we just "Initialize the component"